data/hails.txt

```diff
@@ -3112,6 +3112,64 @@ phrase "hostile militia"
 		"."
 		"!"
 
+phrase "hostile militia"
+	word
+		""
+		"You! "
+		"You there! "
+		"Hey, you! "
+	word
+		"I "
+		"We "
+	word
+		"hope "
+		"wonder if "
+	word
+		"you "
+		"criminals like you "
+		
+	word
+		"like"
```

Comment:
-> `"like "`

Reply:
Oh, I see your point now. How didn't I catch that??

Reply:
Same way everyone else doesn't catch mistakes, too many lines for mistakes to slip into. This is why we review everything.